src/Native/configure.cmake

```diff
@@ -5,11 +5,31 @@ include(CheckIncludeFiles)
 include(CheckPrototypeDefinition)
 include(CheckStructHasMember)
 include(CheckSymbolExists)
+include(CheckTypeSize)
 
 #CMake does not include /usr/local/include into the include search path
 #thus add it manually. This is required on FreeBSD.
 include_directories(SYSTEM /usr/local/include)
 
+# in_pktinfo: Find whether this struct exists
+    check_include_files(
```

Comment:
I couldn't find a better way to make it look related (although I have start-end comments around it). The vetted cmake way would be to have a separate file, something like `in_pktinfo.capability.cmake` in a modules directory, then configure.cmake can call all capabilities in one place which is finally called by CMakeList.txt. 

Perhaps this refactoring can be done at later point if it makes sense in this scenario (where we probably only require finite number of tests in a foreseeable future?). Meanwhile, I will fix the indentation. Thanks. :)